In the module.json, the registry and repo-name must be added

sandervandevelde opened the pull request:
The new module.json has this <registry>/<repo-name>. The step where these values must be added was missing.

I added only point 2 and 3. The other changes are just updating the numbered list.

azurecla commented:
Hi __@jurajsucik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@Dumagar - Please review this pull request in the public repository.

Dumagar commented:
#sign-off

sandervandevelde committed:
In the module.json, the registry and repo-name must be added

The new module.json has this <registry>/<repo-name>. The step where these values must be added was missing

sandervandevelde committed:
Updated reference to module.json

I updated the text so changing the module.json file is now optional.
